.gitignore

```diff
@@ -23,8 +23,12 @@ gfx/pokemon/*/frames.asm
 *.map
 *.sym
 
+# local tools
+rgbds/
+
 # tool binaries
 *.exe
+*.dll
```

Comment:
```suggestion
```
This is not necessary if you are ignoring the `rgbds/` directory.

Reply:
Quite true. But if you install locally on Windows, there are .dll files included with the rgbds pre-compiled downloads that are needed. And if you don't have a global version installed, those are needed.

If you store that version rgbds in a folder such as `pokecrystal/rgbds-v0.5.2/` like INSTALL.md already explains, the .dll files will be picked up.

Reply:
No, the pokecrystal repo does not produce any `*.dll` files, so they should not be in our `.gitignore`. If the user wants to install rgbds *inside* the pokecrystal repo they can smiply add `rgbds/` to their `.git/info/exclude`.

Reply:
Furthermore, rbgds is a seperate repo.. They could add any new files they want to anytime, if it helps them with their goals.

Reply:
I think you are correct with everything you've said so far. But...

> No, the pokecrystal repo does not produce any `*.dll` files, so they should not be in our `.gitignore`. If the user wants to install rgbds _inside_ the pokecrystal repo they can smiply add `rgbds/` to their `.git/info/exclude`. (Or similarly rgbds-v0.5.2/)

At bare minimum, I believe rgbds/ should be added to the .gitignore, or the instructions to use a local version of rgbds should be removed from the INSTALL.md instructions.

These two things are in contradiction. If you follow the approved instructions, you will have things in your repo structure not caught by the default ignore rules.